megalut/plot/scatter.py

```diff
@@ -128,10 +130,12 @@ def scatter(ax, cat, featx, featy, featc=None, cmap="jet", title=None, text=None
 			ax.errorbar(data[featx.colname], data[featy.colname], xerr=xerr, yerr=yerr, **myerrorbarkwargs)
 		
 		stuff = ax.scatter(data[featx.colname], data[featy.colname], c=data[featc.colname], **mykwargs)
-		divider = make_axes_locatable(ax)
-		cax = divider.append_axes("right", "5%", pad="3%")
-		cax = plt.colorbar(stuff, cax)
-		cax.set_label(featc.nicename)
+		
+		if hidecbar is False:
```

Comment:
Not sure, but I could merge fiducial once more, I'll make sure it's up to date before PL-ing